js/index.js

```diff
@@ -10,7 +10,6 @@ search.addEventListener("click", () => {
     request.onload = () => {
         if (request.status >= 200 && request.status < 400) {
```

Comment:
Why req.status < 400?

Reply:
Because errors carry a 400 and above status. But from 400 to <500 means there's an error in the request. so maybe I should use another if statement for that and alert another error message.

Reply:
Added a new conditional statement to catch the different status of request.